libs/template/templates/default-python/template/{{.project_name}}/setup.py.tmpl

```diff
@@ -10,11 +10,12 @@ from setuptools import setup, find_packages
 import sys
 sys.path.append('./src')
 
+import datetime
 import {{.project_name}}
 
 setup(
     name="{{.project_name}}",
-    version={{.project_name}}.__version__,
+    version=datetime.datetime.utcnow().strftime("%Y%m%d.%H%M%S"),
```

Comment:
Could you add a comment here? And shouldn't this be something like `f"1.0.0+{datetime....}"` or `f"1.0.{datetime....}`?